tests/testing_common.py

```diff
@@ -63,6 +65,8 @@
 from .testing_utils import get_state_dict
 
 
+HUB_MODEL_ACCESSES = {}
```

Comment:
Could some logic be simplified if this where a defaultdict with 0 default? Theoretically, `False` / `True` would also be enough but I see the value in counting the cache hits.

Reply:
Yes, counting cache hits was my goal and I think it can still be of value. I'm inclined to keep it like that.